### PR TITLE
TC_10.001.06 | Views > Create new view

### PR DESCRIPTION
**Pull request by ArtemRazdymakha**
https://trello.com/c/QMcbKRcy/231-tc1000106-views-create-new-view

### CODE REVIEW

File: src/test/java/school/redrover/ViewTest.java [hunk 1]
```diff
@@ -54,7 +52,7 @@ public void testCreateNewView() {
 
         Assert.assertEquals(getDriver().findElement(By.xpath("//a[@href='/user/admin/my-views/view/Test%20view/']")).getText(), nameView);
     }
-
+    Ignore
```
Comment (VictoriaLema) [inline src/test/java/school/redrover/ViewTest.java:55]: Hi! If you have to ignore a test, it should be annotate with  @Ignore (@ is missing on 55 line)
P.S. Ignore with capital letter

File: src/test/java/school/redrover/ViewTest.java [hunk 2]
```diff
@@ -97,7 +95,21 @@ public void testCreateNewView2() {
...
+        getDriver().findElement(By.className("jenkins-input")).sendKeys("Bob");
+        getDriver().findElement(By.xpath("//*[@id=\"j-add-item-type-nested-projects\"]/ul/li[1]")).click();
+        getDriver().findElement(By.xpath("//*[@id=\"ok-button\"]")).click();
+        getDriver().findElement(By.xpath("//*[@id=\"breadcrumbs\"]/li[1]/a")).click();
+
+        getDriver().findElement(By.xpath("//*[@id=\"job_Bob\"]/td[3]"));
+        Assert.assertEquals(getDriver().findElement(By.xpath("//*[@id=\"job_Bob\"]/td[3]")).getText(),"Bob");
 
+    }
+    Ignore
```
Comment (VictoriaLema) [inline src/test/java/school/redrover/ViewTest.java:112]: If you have to ignore a test, it should be annotate with  @Ignore (@ is missing on 112 line)

File: src/test/java/school/redrover/ViewTest.java (hunk 1 above)
Comment (ArtemRazdymakha) [reply src/test/java/school/redrover/ViewTest.java]: Hey. Done

**Review by KateZzz — COMMENTED** [review]
Looks like it is not a good practice to leave commented code.I thinks you should remove it

File: src/test/java/school/redrover/ViewTest.java (hunk 1 above)
Comment (VictoriaLema) [reply src/test/java/school/redrover/ViewTest.java]: No, you have commented code. If this test fails, you have to annotate it with @Ignore

File: src/test/java/school/redrover/ViewTest.java (hunk 2 above)
Comment (ArtemRazdymakha) [reply src/test/java/school/redrover/ViewTest.java]: vse

**Review by SafronovaOlga — APPROVED** [review]

**Review by Yakubovwk — APPROVED** [review]

**Pull request merged**